Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1

```diff
@@ -189,8 +190,8 @@ try
 }
 catch
 {
-    Write-Verbose $_.Exception.ToString() -Verbose
-    Write-Telemetry "Task_InternalError" $_.Exception.Message
+    Write-Verbose $_.Exception.ToString()
+    Write-Telemetry "Task_InternalError" "PowerShellOnTargetMachineTaskFailed"
```

Comment:
Can you please publish exception type here in this format PowerShellOnTargetMachineTaskFailed: exceptionType ? This will be more helpful to debug the issue .